controllers/account/controllers/namespace_controller.go

```diff
@@ -2750,7 +2751,9 @@ func deleteResourceAndWait(
 	namespace, name string,
 ) error {
 	deletePolicy := v12.DeletePropagationForeground // Foreground deletion, wait for child resources
-
+	if gvr.Resource == "backups" {
```

Comment:
The `deleteBackup` function needs to be handled, as the backup processing logic was previously moved out.